docs/variants/msi_z690/development.md

```diff
@@ -0,0 +1,76 @@
+# Dasharo compatible with MSI PRO Z690-A WIFI DDR4 - development
+
+## Intro
+
+This document gather various notes and documentation useful for development of
+Dasharo compatible with the MSI PRO Z690-A WIFI DDR4 platform.
```

Comment:
@miczyg1 at least please mention RTE and give some reference where to get it e.g. link to shop or official 3mdeb website or to Dasharo docs section related to RTE. TBH we should have one single point to reference that works for all. I believe this should work: https://docs.dasharo.com/transparent-validation/rte/introduction/

Reply:
We should also refer to sonoff

Reply:
added